README.md

```diff
@@ -1,4 +1,64 @@
-xbundle
-=======
+# xbundle
 
-xbundle tool for edX courses
+`xbundle` converts back and forth between OLX and "xbundle" style
+XML formats. The xbundle format is a single XML file.
+
+The OLX format is defined in [this documentation](http://edx-open-learning-xml.readthedocs.org/en/latest/).
+
+## Installation
+
+`python setup.py install`
+
+This will install `xbundle` and the `xbundle_convert`
+command-line tool.
+
+---
+
+## Using xbundle in your code
+
+### To convert from xbundle to OLX
+
+```python
+    from xbundle import XBundle
+
+    bundle = XBundle()
+    # get input_path and output_path from user input 
+    bundle.load(input_path)
+    bundle.export_to_directory(output_path)
+```
+
+### To convert from OLX to xbundle
+
+```python
+    from xbundle import XBundle
+
+    bundle = XBundle()
+    # get input_path and output_path from user input 
+    bundle.import_from_directory(input_path)
+    bundle.save(output_path)
+```
+
+---
+
+## Options
+
+TODO: Better documentation.
```

Comment:
We should just remove this, and make it an issue, it is easier to track that way and doesn't require a commit/PR to change our todos.  While we are at, we should make issues for pep8 (168 violations), pylint (2.55/10), adding py.test, and adding travis/coverage.